### DISCUSSION TITLE
Question regarding the new task bar

### DISCUSSION THREAD

**Discussion by Gunhedo** [valinet/ExplorerPatcher#3482] [General]
Hi,

I've seen the latest pre-release and it states that it's available for OS: 19045.4598, 22621.3296, 22621.3810, 26120.961, and 26244.5000. (Note: 22621 and 22631 share the same OS files).

I'm on OS: 26100.1150 and my question is, is there a reason why it's not available for versions below 26120.961?

**Comment by pyrates999** [comment]
I think this was already answered here in your previous discussion that you started: https://github.com/valinet/ExplorerPatcher/discussions/3445

**Comment by Amrsatrio** [comment]
@Gunhedo That means it's the versions EP is confirmed to work on. EP is developed with updates (future OS versions) in mind. Doesn't mean that EP or its new taskbar is available only for those versions.
